examples/calculator/lib/main.dart

```diff
@@ -0,0 +1,261 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+
+import 'calc_expression.dart';
+
+// A calculator application.
+void main() {
+  runApp(new MaterialApp(
+      title: 'Calculator',
+      theme: new ThemeData(primarySwatch: Colors.blue),
+      home: new Calculator()));
```

Comment:
nit: if you have a newline after the `(`, there should be a newline before the `)`.